### ISSUE TITLE
nothing happens when running `npm run lint` command

### ISSUE THREAD

**Issue by shootermv** [hasadna/open-bus-map-search#436]
it looks like command doing nothing
here is what happens on my terminal:
![image](https://github.com/hasadna/open-bus-map-search/assets/1336862/2aaab876-dfc6-4b26-a083-66ca399297c9)
instead - it should display lint errors or warnings

**Comment by yardenmezi** [comment]
additional information:
when there is prettier problem, it seems right:
```

> open-bus-map-search@1.0.0 lint
> eslint . --max-warnings 0


.../OperatorSelector.tsx
  24:33  error  Delete `··`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```
Also, when I check logs I get:
`
> open-bus-map-search@1.0.0 lint
> eslint . --max-warnings 0

npm verb exit 0
npm info ok 
`

(Possibly it looks a little weird because not a lot of checks are made?)

**Comment by NoamGaash** [comment]
fixed on #449
